google-home-community.groovy

```diff
@@ -1740,6 +1865,42 @@ private executeCommand_setVolume(deviceInfo, command) {
     ]
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private executeCommand_StartStop(deviceInfo, command) {
+    def startStopTrait = deviceInfo.deviceType.traits.StartStop
+    def checkValue
+    if (command.params.start) {
+        checkMfa(deviceInfo.deviceType, "Start", command)
+        checkValue = startStopTrait.startValue
+        deviceInfo.device."${startStopTrait.startCommand}"()
+    } else {
+        checkMfa(deviceInfo.deviceType, "Stop", command)
+        checkValue = { it = startStopTrait.startValue }
```

Comment:
Should this be `{ it != startStopTrait.startValue }`?  As is it seems like a no-op that will always return a truthy value.

Reply:
Yes it should,  thanks for catching that!